src/backend/distributed/connection/remote_commands.c

```diff
@@ -257,6 +252,32 @@ ReportResultError(MultiConnection *connection, PGresult *result, int elevel)
 }
 
 
+#if (PG_VERSION_NUM < 100000)
```

Comment:
Can you add the `/* *INDENT-OFF* */` and `/* *INDENT-ON* */` comments around this so `citus_indent` skips them? Re-copy the PostgreSQL code exactly so it's easier to diff in the future. It's fine if it doesn't match our style so long as it's copied from PostgreSQL.

Reply:
Makes sense